mtp_api/apps/credit/admin.py

```diff
@@ -51,7 +51,7 @@ class PaymentAdminInline(admin.StackedInline):
     extra = 0
     readonly_fields = ('uuid', 'status', 'batch', 'billing_address',)
 
-    def has_add_permission(self, request):
+    def has_add_permission(self, request, obj=None):
```

Comment:
This didn't really have any affect tho because method just returns `False` right?

Reply:
overridden methods should have arguments compatible with their parent. dunno _why_ the parent might get an `obj` and i don't think it was hit in practice, but there'd be an error if somehow obj was provided to, but not accepted by, this method